benefits/core/models.py

```diff
@@ -352,6 +353,14 @@ class TransitAgency(models.Model):
         help_text="The name of the secret containing this agency's client_secret value used to access the TransitProcessor's API.",  # noqa: E501
         default="",
     )
+    group = models.OneToOneField(
```

Comment:
Let's name this something a little more explicit - like `staff_group` or `user_group` or something. 

E.g. we use `EnrollmentFlow.group_id` so this may be confusing.

Reply:
Done in [9b36c7c](https://github.com/cal-itp/benefits/pull/2307/commits/9b36c7c3af566d144cac61c9fca62599b0cdeb13)